spec/features/variable_drafts/forms/dimensions_form_spec.rb

```diff
@@ -28,8 +28,8 @@
       expect(page).to have_selector(:link_or_button, 'Add another Dimension')
     end
 
-    it 'has three required labels' do
-      expect(page).to have_selector('label.eui-required-o', count: 3)
+    it 'has 2 required labels' do
```

Comment:
So for Dimensions and for Sets, you had to reduce the number of required labels by 1, but looking at the schema that seems right. I'm just wondering where that extra required icon/item came from.

Reply:
![screen shot 2018-02-27 at 9 58 01 am](https://user-images.githubusercontent.com/171388/36735718-cece1e18-1ba4-11e8-902e-45e068310cf4.png)
the top level field used to have an icon, it might need the icon still, but that sounds like 1018 can look at it